src/reducers/filter.js

```diff
@@ -0,0 +1,10 @@
+const filter = (state = '', action) => {
+  switch (action.type) {
+    case 'CHANGE_FILTER':
+      return action.filter;
+    default:
+      return state;
+  }
+};
+
+export default filter;
```

Comment:
it should default to `ALL` [[requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)]

Reply:
Thanks a lot. We will fix it.